tests/system/providers/amazon/aws/example_sagemaker.py

```diff
@@ -176,8 +177,8 @@ def _build_and_upload_docker_image(preprocess_script, repository_uri):
         _, stderr = docker_build.communicate()
         if docker_build.returncode != 0:
             raise RuntimeError(
-                "Failed to push docker image to the repository.  The following error "
-                f"message may be useful, but can occasionally be misleading: {stderr}"
+                "Failed to prepare docker image for the preprocessing job.\n"
+                f"The following error happened while executing the sequence of bash commands:\n{stderr}"
```

Comment:
I feel like saying that the message could be misleading _is_ misleading in itself.
Also, "failed to push" is misleading too, since pushing is not the only thing we're doing here, I was actually confused for a while while investigating this why there was a rate limit error on a push operation.